kalaponents.module

```diff
@@ -82,19 +84,19 @@ function theme_paragraphs_item__kalaponent__full(&$vars) {
 
       // Link text and url as separate variables.
       case 'field_kalaponents_link':
-        $variables['link_text'] = $paragraph->$field->url->value();
-        $variables['link_href'] = $paragraph->$field->title->value();
+        $vars['link_text'] = $paragraph->$field->url->value();
```

Comment:
@gremy , what you mean by "crash?" It works fine for me. The only effect of replacing `$variables` with `$vars` I can see is adding in all the Drupal template variables that come with the theme override, which we don't want.

Thanks!